wled00/FX_fcn.cpp

```diff
@@ -712,11 +712,16 @@ void IRAM_ATTR_YN Segment::setPixelColor(int i, uint32_t col)
 {
   if (!isActive()) return; // not active
 #ifndef WLED_DISABLE_2D
-  int vStrip = i>>16; // hack to allow running on virtual strips (2D segment columns/rows)
+  int vStrip;
```

Comment:
I'm not a big fan of leaving locals uninitialized... are you sure that vStrip is never used without first assigning a value?

Reply:
good call.